<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>shownetwork</class>
 <widget class="QWidget" name="shownetwork">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1135</width>
    <height>930</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>500</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>500</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>网络拓扑图</string>
     </property>
     <widget class="BatteryItem" name="widget_9" native="true">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>360</y>
        <width>200</width>
        <height>150</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>150</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>300</height>
       </size>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>1028</y>
        <width>24</width>
        <height>16</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string>充电</string>
      </property>
     </widget>
     <widget class="SlaveItemWidget" name="widget_4" native="true">
      <property name="geometry">
       <rect>
        <x>11</x>
        <y>341</y>
        <width>150</width>
        <height>100</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>100</height>
       </size>
      </property>
     </widget>
     <widget class="SlaveItemWidget" name="widget_2" native="true">
      <property name="geometry">
       <rect>
        <x>11</x>
        <y>129</y>
        <width>150</width>
        <height>100</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>100</height>
       </size>
      </property>
     </widget>
     <widget class="SlaveItemWidget" name="widget_3" native="true">
      <property name="geometry">
       <rect>
        <x>11</x>
        <y>235</y>
        <width>150</width>
        <height>100</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>100</height>
       </size>
      </property>
     </widget>
     <widget class="SlaveItemWidget" name="widget_5" native="true">
      <property name="geometry">
       <rect>
        <x>11</x>
        <y>447</y>
        <width>150</width>
        <height>100</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>100</height>
       </size>
      </property>
     </widget>
     <widget class="SlaveItemWidget" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>11</x>
        <y>23</y>
        <width>150</width>
        <height>100</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>100</height>
       </size>
      </property>
     </widget>
     <widget class="SlaveItemWidget" name="widget_8" native="true">
      <property name="geometry">
       <rect>
        <x>11</x>
        <y>765</y>
        <width>150</width>
        <height>100</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>100</height>
       </size>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="SlaveItemWidget" name="widget_6" native="true">
      <property name="geometry">
       <rect>
        <x>11</x>
        <y>553</y>
        <width>150</width>
        <height>100</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>100</height>
       </size>
      </property>
     </widget>
     <widget class="SlaveItemWidget" name="widget_7" native="true">
      <property name="geometry">
       <rect>
        <x>11</x>
        <y>659</y>
        <width>150</width>
        <height>100</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>100</height>
       </size>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>数据显示图</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>GroupBox</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="GaugeWidgets" name="widget_10" native="true"/>
           </item>
           <item>
            <widget class="GaugeWidgets" name="widget_11" native="true"/>
           </item>
           <item>
            <widget class="GaugeWidgets" name="widget_12" native="true"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout"/>
      </item>
      <item>
       <widget class="QTableWidget" name="tableWidget">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>500</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <column>
         <property name="text">
          <string>新建列</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>新建列</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>新建列</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>新建列</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>新建列</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>新建列</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>新建列</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>新建列</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>新建列</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>新建列</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>新建列</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>新建列</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BatteryItem</class>
   <extends>QWidget</extends>
   <header>src/public/batteryitem.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SlaveItemWidget</class>
   <extends>QWidget</extends>
   <header>src/showslave/slaveitemwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GaugeWidgets</class>
   <extends>QWidget</extends>
   <header location="global">src/shownetwork/GaugeWidgets.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
